HW8(MVC)/scripts/Model.js

```diff
@@ -0,0 +1,37 @@
+// NOT INTERACTING WITH A CONTROLLER. SEEMS LIKE A MISTAKE.
+class Model {
+  constructor() {}
+
+  addTask(title) {
+    const currentTasks = this.getTasksList();
+
+    currentTasks.push({
+      'title': title,
+      'isCompleted': false
+    });
```

Comment:
smearing logic. why can't all this happen in the setTasksList method?

Reply:
1. Function: Adding a new task. Input - title (string)
2. Function: Saving current tasks list - tasksList(array). For example, after deleting tasks.

Improved function names for a better visible difference.